src/config/wallets.ts

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
 import { ReactComponent as PolkadotExtensionLogoIcon } from '@/assets/img/polkadot-extension-logo.svg';
-import { ReactComponent as SubWalletLogoIcon } from '@/assets/img/subwallet-logo.svg';
+import { ReactComponent as SubWalletLogoIcon } from '@/assets/img/sub-wallet-logo.svg';
```

Comment:
Don't think this is right - the wallet is called SubWallet (not Sub), so should be left as a single word